keyboards/massdrop/ctrl/keymaps/endgame/keymap.c

```diff
@@ -215,8 +255,28 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
                 }
             }
             return false;
-        case RGB_TOG:
-            if (record->event.pressed) {
+    }
+
+    if (record->event.pressed) {
+        // The Yakuake config uses LGUI+Ctrl+Shift+<keycode>. KC_NO used for undesired key, all mapped ones get mods.
+        if (biton32(layer_state) == _YL && keycode != TG(_YL)) {
```

Comment:
```suggestion
        if (get_highest_layer(layer_state) == _YL && keycode != TG(_YL)) {
```

Reply:
Cool, why is this better though? aren't they the same?